src/utils/sortPeople.tsx

```diff
@@ -0,0 +1,28 @@
+import { Person } from '../types';
+
+export const sortPeople = (sortParam: string, people: Person[]) => {
+  switch (sortParam) {
+    case 'name':
```

Comment:
[Try not to hardcode these values, but use Enum](https://www.typescriptlang.org/docs/handbook/enums.html) instead